rdmo/management/rules.py

```diff
@@ -85,12 +72,16 @@ def is_legacy_reviewer(user) -> bool:
 
 
 # Add rules
-rules.add_rule('management.can_view_management',
```

Comment:
this will break the backwards compatibility on the `base_navigation` template, should I keep the the `add_rule` in there just to not break the possible deployed custom templates?
Otherwise instance admins would need to update the theme for that. Think that it is better to use `add_perm` than `add_rule`..

Reply:
ok, Ive renamed the permissions like that. The `has_perm` is better for when it will be used on a user object anyway..